packages/adapters/examples/segmentationVolume/index.ts

```diff
@@ -0,0 +1,487 @@
+import * as cornerstone from "@cornerstonejs/core";
+import * as cornerstoneTools from "@cornerstonejs/tools";
+import * as cornerstoneDicomImageLoader from "@cornerstonejs/dicom-image-loader";
+import * as cornerstoneAdapters from "@cornerstonejs/adapters";
+
+import {
+    addButtonToToolbar,
+    addDropdownToToolbar,
+    addManipulationBindings,
+    createImageIdsAndCacheMetaData,
+    createInfoSection,
+    initDemo,
+    labelmapTools,
+    setTitleAndDescription
+} from "../../../../utils/demo/helpers";
+
+import dcmjs from "dcmjs";
+
+// This is for debugging purposes
+console.warn(
+    "Click on index.ts to open source code for this example --------->"
+);
+
+const {
+    Enums: csEnums,
+    RenderingEngine,
+    cache,
+    imageLoader,
+    metaData,
+    setVolumesForViewports,
+    utilities: csUtilities,
+    volumeLoader
+} = cornerstone;
+const { ViewportType } = csEnums;
+
+const {
+    Enums: csToolsEnums,
+    SegmentationDisplayTool,
+    ToolGroupManager,
+    segmentation
+} = cornerstoneTools;
+const { MouseBindings } = csToolsEnums;
+
+const { wadouri } = cornerstoneDicomImageLoader;
+
+const { adaptersSEG, helpers } = cornerstoneAdapters;
+const { Cornerstone3D } = adaptersSEG;
+const { downloadDICOMData } = helpers;
+
+//
+const volumeName = "CT_VOLUME_ID";
+const volumeLoaderScheme = "cornerstoneStreamingImageVolume";
+const volumeId = `${volumeLoaderScheme}:${volumeName}`;
+
+let renderingEngine;
+const renderingEngineId = "MY_RENDERING_ENGINE_ID";
+const toolGroupId = "MY_TOOL_GROUP_ID";
+const viewportIds: string[] = ["CT_AXIAL", "CT_SAGITTAL", "CT_CORONAL"];
+let imageIds: string[] = [];
+
+// ======== Set up page ======== //
+
+setTitleAndDescription(
```

Comment:
Great PR!

I'm not sure what is the difference between `segmentationVolume` and `segmentationVolumeLocal` examples, can you please update each description to make it evident? Thanks

Reply:
`segmentationVolumeLocal` replace for `segmentationVolume`, just `segmentationVolume`

Reply:
The two are similar, when it says "local" it is to import dicom, now everything together in one.